package.json

```diff
@@ -22,10 +22,9 @@
     "setup:dev": "ts-node ./script/setup.ts",
     "docker:dev": "docker-compose -f docker-compose.dev.yml up -d"
   },
-  "dependencies": {
+   "dependencies": {
```

Comment:
looking good! just undo this space please

Reply:
space before last "dependencies":?

Reply:
yes :)

Reply:
i have undo the space

Reply:
why it is showing outdated?